lib/fastlane/plugin/wpmreleasetoolkit/models/firebase_test_lab_result.rb

```diff
@@ -0,0 +1,42 @@
+module Fastlane
+  class FirebaseTestLabResult
+    def initialize(log_file_path:)
+      raise "No log file found at path #{log_file_path}" unless File.file? log_file_path
+
+      @path = log_file_path
+    end
+
+    # Scan the log file to for indications that the Test Run failed
+    def success?
+      File.readlines(@path).none? { |line| !line.include? 'Failed' }
```

Comment:
💥 Double negation in "None of the lines don't include" feels a bit hard to parse and reason about… and in fact seems wrong to me and the opposite of what we want?

We should consider that we have a **success** if **none** of the line **include** "Failed", right?

```suggestion
      File.readlines(@path).none? { |line| line.include? 'Failed' }
```

Reply:
cc @oguzkocer FYI as you're testing this PR with a real-world usage in a client app, and this code/issue might lead to you seeing results opposite to what you'd expect

Reply:
Excellent catch there – this is addressed in ff89bd09a552bfebe2831404aaa9dcb4329f3243.